code/style.css

```diff
@@ -168,66 +257,85 @@ button {
   text-align: center;
 }
 
-.win-or-lose img {
+.win-or-lose img, .start-game img {
   align-self: flex-start;
   margin-left: 15%;
 }
 
-.win-or-lose h1 {
+.win-or-lose h1, .start-game h1 {
   margin-bottom: 32px;
 }
 
 @media (max-width: 768px) {
   body {
     flex-direction: column;
   }
-
   .question-section {
     width: 100%;
     min-height: 0;
   }
   button {
     padding: 8px 16px;
   }
+  .guess-count {
+    margin-left: 1.5rem;
+    font-size: 18px;
+  }
+  .timer {
+    width: 100px;
+    font-size: 18px;
+  }
+  .restart-button {
+    margin-left: 3rem;
+  }
+  .filled-button, .find-out, .restart-button {
+    font-size: 18px;
+  }
   h1 {
     font-size: 24px;
     line-height: 30px;
   }
-
+  select {
+    font-size: 20px;
+  }
+  option {
+    font-size: 20px;
+  }
   .guess-who-icon {
     width: 51px;
   }
-
+  .created-by-logo {
+    display: none;
+  }
   .board-wrapper {
     width: 100%;
   }
-
   .game-board {
     width: 100%;
     max-width: 750px;
+    justify-content: center;
   }
-
   .card {
-    width: 113px;
+    width: 94px;
     position: relative;
   }
-
+  .card p {
+    font-size: 15px;
+  }
   .card .guess {
     display: flex;
     top: 0;
     position: absolute;
     justify-content: flex-end;
   }
-
   .card .guess span {
     display: none;
   }
-
   .card .guess .filled-button {
     padding: 6px 11px;
     margin-bottom: 1px;
     background: white;
     color: var(--primary);
     border: none;
   }
-}
+}
```

Comment:
Maybe add a media query for sizes from 769? You lose part of the guesses part and restart button.